### PR TITLE
update ingestion tasks

### PR DESCRIPTION
**Pull request by HumnaAleem**

### CODE REVIEW

File: elixir/lib/mix/tasks/platform.ex [hunk 1]
```diff
@@ -21,22 +21,24 @@ defmodule Mix.Tasks.Platform do
 		|> Enum.map(fn row -> row end)
 
 		diagnostic_test_obj = diagnostic_test
-		|> Enum.reduce(%{}, fn([test_id, question_id, question_text, answer, slo]), agg ->
+		|> Enum.reduce(%{}, fn([test_id, question_id, question_text, correct_answer, grade_level, slo_category, slo]), agg ->
 			result = %{
 				"question_text" => question_text,
-				"answer" => answer,
+				"correct_answer" => correct_answer,
```
Comment (taimur38) [inline elixir/lib/mix/tasks/platform.ex:27]: you don't need to change the variable name here.... you will then need to change it everywhere on the frontend also

File: elixir/lib/mix/tasks/platform.ex [hunk 2]
```diff
@@ -21,22 +21,24 @@ defmodule Mix.Tasks.Platform do
...
 
 		assessments_obj = assessments
-		|> Enum.reduce(%{}, fn([test_id, subject, grade, label, type, pdf_url]), agg ->
+		|> Enum.reduce(%{}, fn([test_id, subject, grade_level, label, type, pdf_url]), agg ->
 
 			test = %{
 					"label" => label,
 					"subject" => subject,
-					"grade" => grade,
+					"grade_level" => grade_level,
```
Comment (taimur38) [inline elixir/lib/mix/tasks/platform.ex:41]: same here... we don't need to change our variable names

File: elixir/lib/mix/tasks/platform.ex [hunk 3]
```diff
@@ -83,14 +85,16 @@ defmodule Mix.Tasks.Platform do
...
 				"subject" => subject,
-				"video_links" => video_links,
-				"pdf_link" => pdf_link
+				"lesson_number" => lesson_number,
+				"lesson_title" => lesson_title,
+				"lesson_duration" => lesson_duration,
+				"lesson_link" => lesson_link,
+				"material_links" => material_links,
+				"activity_links" => activity_links
+				"teaching_manual_link" => teaching_manual_link
```
Comment (taimur38) [inline elixir/lib/mix/tasks/platform.ex:97]: fine to change variable names but now need to change it on the frontend.

**Review by taimur38 — CHANGES_REQUESTED** [review]

File: elixir/lib/mix/tasks/platform.ex (hunk 3 above)
Comment (Mudassar045) [inline elixir/lib/mix/tasks/platform.ex:97]: ending `comma` is missing on line 96. While compiling, Elixir should through an error for this.

**Review by taimur38 — APPROVED** [review]

**Pull request merged**